src/components/recipes/RecipeDetails.js

```diff
@@ -107,7 +107,7 @@ export class ArgumentsValue extends React.PureComponent {
   render() {
     const { name, value } = this.props;
 
-    let valueRender = x => x;
+    let valueRender = x => (typeof x === 'object' ? JSON.stringify(x, null, 2) : x);
```

Comment:
I don't really know how this works but if you use the "Copy to clipboard" button, the JSON that is copied to the clipboard does not have the same pretty-print. Can we fix that?

Reply:
yes!